### PR TITLE
[Hdf] cleaning tests

### PR DESCRIPTION
**Pull request by philbucher**
this PR cleans a bit the tests and removes deprecated code
Also introduces the testing-script for the mpi-tests

### CODE REVIEW

File: applications/HDF5Application/python_scripts/create_xdmf_file.py [hunk 1]
```diff
@@ -2,8 +2,13 @@
...
 import KratosMultiphysics.HDF5Application as KratosHDF5
-import os, sys, h5py, xdmf
-
+import os, sys, xdmf
+import warnings
+with warnings.catch_warnings():
+    # suppressing an import-related warningfrom h5py
+    # problem appears when using it in a test with python >=3.6
+    warnings.simplefilter('ignore', category=ImportWarning)
+    import h5py
```
Comment (philbucher) [inline applications/HDF5Application/python_scripts/create_xdmf_file.py:11]: when using h5py in the tests I get a warning:
~~~
/usr/lib/python3.6/importlib/_bootstrap.py:219: ImportWarning: can't resolve package from __spec__ or __package__, falling back on __name__ and __path__
     return f(*args, **kwds)
~~~
See here: https://stackoverflow.com/questions/51045319/how-to-suppress-importwarning-in-a-python-unittest-script

**Comment by philbucher** [comment]
ping

**Comment by philbucher** [comment]
ping, this is just cleanup/removing deprecated code

**Review by msandre — APPROVED** [review]
:+1:

**Pull request merged**